### ISSUE TITLE
Request for enhancement

### ISSUE THREAD

**Issue by rhoddan** [niklaswall/node-red-contrib-light-scheduler#31]
It would be great to have possibilty to support ”special red days”. For example 1st of May in Sweden is similar as Sunday...

...and thanks for a great node!

**Comment by niklaswall** [comment]
Yes, that's on the list... It's pretty easy to create an "override list" that says, treat 1st of May as a Sunday.

BUT, it would be nicer to have it automatically understand that day X is a Sunday, so therefor day X-1 is a "Saturday". And then have that fetched from the cloud so one can crowd-source the exceptional days for a specific region without the need to manually define them.

Step one will of course be to create a manual override list :-)

/Niklas
